oar/core/config_store.py

```diff
@@ -195,6 +195,18 @@ def get_jira_server(self):
         """
         return self._local_conf["jira_server"]
 
+    def get_jenkins_server(self):
+        """
+        Get jenkins server url
+        """
+        return self._local_conf["jenkins_server"]
+
+    def get_jenkins_username(self):
+        """
+        Get jenkins username
+        """
+        return self._local_conf["jenkins_user"]
```

Comment:
jenkins user/token should be consistent, suggest to declare user as env var as well.

Reply:
updated